frontend/src/components/AddConcepts.tsx

```diff
@@ -0,0 +1,134 @@
+import React, {useState, useEffect} from 'react';
```

Comment:
## Remove `Graph` route for now?
<img width="290" alt="Screenshot 2024-04-26 at 6 32 43 PM" src="https://github.com/jhu-bids/TermHub/assets/13045020/2c2c552c-2d9e-476d-98ef-2e7dbb5c5856">